projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -8,7 +8,7 @@ import { MenuItemComponent } from './menu-item/menu-item.component';
   styleUrls: ['./menu-dropdown.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="menu-dropdown">
+    <ht-event-blocker event="click" class="menu-container" [enabled]="this.disabled">
```

Comment:
now both parent and child elements have `menu-container` class on them. The parent should be `menu-dropdown`